app/src/components/fields/fieldErrorHint/fieldErrorHint.jsx

```diff
@@ -246,16 +247,20 @@ export class FieldErrorHint extends Component {
     error: '',
     active: false,
     staticHint: false,
+    staticTouchedHint: false,
     widthContent: false,
     darkView: false,
-    provideHint: true,
```

Comment:
Please discuss with @chivekrodis.
Does removing this prop break anything on new project settings?

Reply:
Maybe developing was with not the last changes?

Reply:
Look like, but I've made new branch from develop after fetch, I'll check